firmware/ec/Makefile

```diff
@@ -78,6 +78,10 @@ LLIBS += -Wl,--end-group
 
 OBJCOPY = $(TOOLCHAIN)/bin/arm-none-eabi-objcopy
 
+default:
```

Comment:
Can you update the Makefiles so that there is a dependency build instead of only happening on default? i.e. make MAIN_OBJS include auto_schema.o, so the rule to build auto_schema.c must be executed.

Reply:
I will invoke script before MAIN_OBJS is initialized.  This guarantees that auto_schema.c is always included.